src/Player/PlayerStrategies.cpp

```diff
@@ -249,16 +266,17 @@ namespace Player {
         num_of_armies.shrink_to_fit();
         int selected_num_armies = Terminal::print_select(num_of_armies);
         selected_num_armies++;
+        player->set_number_armies(selected_num_armies);
 
         source_country->set_armies(source_country->get_armies() - selected_num_armies);
         target_country->set_armies(target_country->get_armies() + selected_num_armies);
         ostringstream os;
         os << selected_num_armies << " armies have been transferred from " << source_country->get_name() << " to " << target_country->get_name();
         Terminal::print(os.str());
 
-        fortificate_country.emplace_back(source_country);
-        fortificate_country.emplace_back(target_country);
-        return fortificate_country;
+//        fortificate_country.emplace_back(source_country);
+//        fortificate_country.emplace_back(target_country);
+        return fortificate_country; // make sure this is empty
```

Comment:
Not sure to understand this... as there is logic outside of this method that uses the return values.

Reply:
To maintain fluid view updating, if it is a human player, updates will be done within this method (first source country, then target, both result in an update), but if it is a computer player, update will occur in the Player.cpp fortify method, and will only happen after both the source country and target country are set. This will be less choppy.

Reply:
So.. you always return an empty array? Why not refactoring the method to return void instead? And remove commented code please.

Reply:
Only from the Human strategy, all other strategies return a 2 member vector

Reply:
Gotcha!